content/page/getting-involved/index.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Getting involved
+menu:
+    main: 
+        weight: 5
+        params:
+            icon: code-icon
+
+comments: false
+---
+
+The easiest way to get in touch is probably through [Discord](https://discord.gg/C9bzWgNmqk), where we discuss research topics, job opportunities, open hardware, spiking neural network training and much more. We'd be delighted to have you join! 
+If you feel like contributing to ONM but you're not exactly sure how, here are some ideas to get you started:
+
+* Link an interesting open source repository to our [collection](https://github.com/open-neuromorphic/open-neuromorphic) so that others can find it too! This can be a framework / package that deals with neuromorphic things or a neat implementation for example! ONM is meant as a platform to showcase your code and others'!
+* Write a blog post together with the community. If you think you learned something useful that you'd like to share with the community, why not publish it on our website? You can have a look at the structure of [existing posts](https://github.com/open-neuromorphic/open-neuromorphic.github.io/tree/main/content/post) and open a PR for a new one. The community will then give you feedback to improve it if you wish to do so. Think about it as a mini paper!
+* Why not host your repository under ONM directly? You can create a new repository to start working on an idea that hopefully benefits the community, or migrate an existing repository. That will make you a member of ONM automatically!
```

Comment:
And I will also add that it would get visibility and help from experienced open sourcers, or something like that

Reply:
how about the new version?

Reply:
I love it!